### ISSUE TITLE
Tests: "should render on server" fails

### ISSUE THREAD

**Issue by timwis** [choojs/choo#526]
```
    expected: '<p>Hello filthy planet</p>'
    actual:   '\n      <p>Hello filthy planet</p>\n    '
```
A `.trim()` in the test would resolve this (or using `html` without the whitespace) but wanted to check if this was intended to pass as-is before doing that.

**Comment by tgfjt** [comment]
related?
https://github.com/shuhei/pelo/issues/7

**Comment by yoshuawuyts** [comment]
^ yeah, that's the reason - we should update pelo I reckon

**Comment by jbergstroem** [comment]
I just filed https://github.com/choojs/choo/pull/532; guessing that's the incorrect approach then?

**Comment by goto-bus-stop** [comment]
seems fixed by #532
